Vizzuality/foodscapes
# setup for Datasette service [FOOD-76] [FOOD-63] [FOOD-64] [FOOD-98]

This PR allows to spin up a Datasette instance with Foodscapes tabular data via the single Docker Compose setup for the whole set of services.

It also includes the change needed to let the Docker build process for the container images of both the TiTiler and Datasette services retrieve and bake the source data (COG file and SQLite database) into the respective container images.

And finally, I sneaked in a small change to allow easily configuring tiler and datasette services to be served behind a reverse proxy, as we need it for the initial dev environment setup (and this could possibly be useful in staging/production depending on the specific setup we decide on).

## How to test

- Ideally, make sure your local Docker environment doesn't have any traces of previous Foodscapes Docker setup
- Make sure your Docker is up to date (see main `README.md` about minimum recommended version and specific setting to activate/update in case BuildKit builds were not enabled by default in previous local installs of Docker)
- Following the _Quick start_ section of the `README` should build and start the client app, the TiTiler service and Datasette

For the scope of this PR, Datasette should then be available at http://localhost:3203 (if using the default port settings from `env.default` copied over to `.env`.
